hack/cluster-configs/config-dpu.yaml

```diff
@@ -11,10 +11,16 @@ clusters:
       kind: "dpu"
       bmc:
         url: "{{bmc_hostname(0)}}"
-      host_side_bmc: "{{bmc(0)}}"
+      host_bmc:
+        user: "root"
+        password: "calvin"
+        url: "{{bmc(0)}}"
       dpu_host: "{{worker_name(0)}}"
       ip: "172.16.3.16"
       mac: "{{DPU_mac_address(0)}}"
+    preconfig:
+    - name: dpu_firmware
+      mev_version: "1.8.0.10052"
```

Comment:
This setting makes no sense for Marvell DPU. This configuration should to work also with Marvell DPU.

Reply:
that's why we have `mev_version` and not `dpu_version`. The code will skip marvell untill we implement that part.